third_party/ibis/ibis_postgres/client.py

```diff
@@ -99,7 +99,13 @@ def _get_type(typestr: str) -> dt.DataType:
     else:
         typestr_wo_length = typestr_wob
     typ = _type_mapping.get(typestr_wo_length)
-    if typ is not None:
+    # Added != "numeric" below for issue:
+    #   https://github.com/GoogleCloudPlatform/professional-services-data-validator/issues/1302
+    # We do not want to map all numerics to decimal(None, None), we want "numeric" to be
+    # passed to the _parse_numeric() function.
+    # An alternative was to remove "numeric" from _type_mapping but that would be yet more monkey
+    # patching, at least this function is already patched.
+    if typ is not None and typestr_wo_length != "numeric":
```

Comment:
Is there any Postgres doc we can point to regarding `typestr_wo_length` != "numeric"` or is this something from DVT/ibis

Reply:
The value comes from the PostgreSQL catalog and is referenced as a string in Ibis code:
```
_type_mapping = {
    "bigint": dt.int64,
    "boolean": dt.bool,
    "bytea": dt.binary,
    "character varying": dt.string,
    "character": dt.string,
    "character(1)": dt.string,
    "date": dt.date,
    "double precision": dt.float64,
    "geography": dt.geography,
    "geometry": dt.geometry,
    "inet": dt.inet,
    "integer": dt.int32,
    "interval": dt.interval,
    "json": dt.json,
    "jsonb": dt.json,
    "line": dt.linestring,
    "macaddr": dt.macaddr,
    "macaddr8": dt.macaddr,
    "numeric": dt.decimal,
    "point": dt.point,
    "polygon": dt.polygon,
    "real": dt.float32,
    "smallint": dt.int16,
    "text": dt.string,
    # NB: this isn't correct because we're losing the "with time zone"
    # information (ibis doesn't have time type that is time-zone aware), but we
    # try to do _something_ here instead of failing
    "time with time zone": dt.time,
    "time without time zone": dt.time,
    "timestamp with time zone": dt.Timestamp("UTC"),
    "timestamp without time zone": dt.timestamp,
    "uuid": dt.uuid,
}
```
I don't find any of this code in recent Ibis versions so this might all go away if/when we upgrade.